docs/api.rst

```diff
@@ -14,22 +14,25 @@ Simulation model
 
 The simulation model in ``msprime`` closely follows the classical ``ms``
 program. Unlike ``ms``, however, time is measured in generations rather than
-"coalescent units". Internally the same simulation algorithm is used, but
-``msprime`` provides a translation layer to allow the user input times and
-rates in generations. Similarly, the times associated with the trees produced
```

Comment:
I thought it was unnecessary and confusing to mention the under-the-hood time scaling, since the user never sees this.